chapters/530_access-management.md

```diff
@@ -63,10 +63,10 @@ further attention from the *operator*, or it can start to process and create the
 *access request* and also on what a *permission profile* determines (see 'types of access requests'
 below).
 
-3)  __Respond to consumer__; handover the computed response back to the requester by proceeding with 
-one of the two following options. Either the system responses with a current status of the process 
+3)  __Respond to consumer__; handover of the computed response back to the requester by proceeding with 
```

Comment:
"handover of the computed response" sounds a bit bumpy to me, because "handover" was meant to be a verb